en/mkdocs.yml

```diff
@@ -868,7 +868,7 @@ nav:
               - SOAP Headers: reference/mediators/property-reference/soap-headers.md
               - Axis2 Properties: reference/mediators/property-reference/axis2-properties.md
               - Synapse Message Context Properties: reference/mediators/property-reference/message-context-properties.md
-              - Accessing Properties with XPath: reference/mediators/property-reference/accessing-properties-with-xpath.md
+          - Expressions: reference/synapse-properties/expressions.md
```

Comment:
Since we don't have the file `reference/synapse-properties/expressions.md` in the 4.2.0 documentation, add the relevant redirection to the **4.2.0** branch to avoid getting the 404 page when switching between the versions.